pom.xml

```diff
@@ -823,6 +823,17 @@
                 <artifactId>maven-enforcer-plugin</artifactId>
                 <version>3.1.0</version>
                 <executions>
+                    <execution>
+                        <id>no-duplicate-declared-dependencies</id>
+                        <goals>
+                            <goal>enforce</goal>
+                        </goals>
+                        <configuration>
+                            <rules>
+                                <banDuplicatePomDependencyVersions/>
```

Comment:
Instead of adding a new `execution` block, the `banDuplicatePomDependencyVersions` rule should be added to the existing `execution` section.